src/core/highlight-vars.js

```diff
@@ -0,0 +1,83 @@
+// Module core/highlight-vars
+// Highlights occurrences of a <var> within a section on click
+// if a <var> is set only once (unused/undeclared), it's highlighted with class ".bug"
+// removes highlights from <var> if clicked anywhere else
+// all is done while keeping in mind that exported html stays clean (removeHighlight)
+
+import hlVars from "deps/text!core/css/var.css";
+const styleElement = document.createElement("style");
+styleElement.textContent = hlVars;
+styleElement.classList.add("removeOnSave");
+document.head.appendChild(styleElement);
+
+export const name = "core/highlight-vars";
+
+export function run(conf, doc, cb) {
+  document.addEventListener("click", initHighlight);
+  cb();
+}
+
+// availability of highlight colors.
+const HL_COLORS = {};
+[ // background[,color]
+  "yellow",
+  "#726012,#fff",
+  "#c78e00",
+  "#3949ab,#fff",
```

Comment:
the first one is `background`, the second one is for `color`.
If the second one is not given, it'll be default color (black). this is done so that we have good color contrasts